src/domain/collection/collectionHero/CollectionHero.tsx

```diff
@@ -40,7 +42,8 @@ const CollectionHero: React.FC<Props> = ({ collection }) => {
         {!!linkText && !!linkText && (
```

Comment:
seems like here is some old code that has !!linkText twice, could be removed now